poc_config/openmrs/apps/registration/app.json

```diff
@@ -7,15 +7,16 @@
                 { "name":"Estado Civil", "display":true }
             ],
             "autoCompleteFields":["familyName", "caste"],
-            "defaultIdentifierPrefix": "GAN",
-            "defaultVisitType": "OPD",
-            "searchByIdForwardUrl": "/patient/{{patientUuid}}",
             "showMiddleName": false,
             "encounterTypes" : {
```

Comment:
If all those uuids were put in a different file(say constants) and be accessed elsewhere by name, it would be easier to use and maintain especially if someone has to change their value and maybe they have been used many times in several other places